.github/settings.yml

```diff
@@ -76,9 +80,7 @@ branches:
       required_status_checks:
         strict: false
         contexts:
-          - rustfmt
-          - markdown-lint
-          - yamllint
```

Comment:
are these not required anymore?

Reply:
These three jobs are now just steps of 'lint', since none of them takes more than a couple seconds to run.